src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -496,6 +496,11 @@ internal virtual IOperation GetOperationWorker(CSharpSyntaxNode node, GetOperati
         public abstract SymbolInfo GetSymbolInfo(SelectOrGroupClauseSyntax node, CancellationToken cancellationToken = default(CancellationToken));
 
         /// <summary>
+        /// Gets the symbol information for the property of a sub-property pattern.
+        /// </summary>
+        public abstract SymbolInfo GetSymbolInfo(SubPropertyPatternSyntax node, CancellationToken cancellationToken);
```

Comment:
Fine, though there are no clients that would use it. The enclosing class is internal.